resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,60 @@
+@extends('layouts.layout')
+
+@section('content')
+    <div>
```

Comment:
![image](https://user-images.githubusercontent.com/101800635/230720465-356071cc-043b-4fa7-bd23-41be427586c1.png)
Officieel is de contrast van de sterretjes net niet hoog genoeg. Kan snel worden opgelost door de sterretjes een klein beetje donkerder te maken

Reply:
fixed